<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>settingDiag</class>
 <widget class="QDialog" name="settingDiag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>301</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>301</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>設定</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>水深</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X方向速度分量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y方向速度分量</string>
   </property>
  </widget>
  <widget class="QComboBox" name="depthUnit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>cm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ft.</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>in.</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="xDirUnit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>m/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>in/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>in/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ft/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ft/min</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="yDirUnit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>m/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cm/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>in/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>in/min</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ft/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ft/min</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>單位</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PMingLiu</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>解析度</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="depthAttrCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="xDirAttrCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="yDirAttrCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>271</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>settingDiag</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>settingDiag</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
